cypress/e2e/signup.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('', () => {
+    it('Can sign up with valid data', () => {
+        cy.visit("/")
+        cy.get('a.nav-link').contains('Sign up').click()
+        cy.get('input[placeholder="Username"]').type('Wrtey6123')
+        cy.get('input[placeholder="Email"]').type('Wrtey6123@mail.com')
+        cy.get('input[placeholder="Password"]').type('testUser123')
+        cy.get('button[type="submit"]').click()
```

Comment:
Add an assertion to check if the username appears in the site navigation after signing in.